website/docs/r/cloudtrail.html.markdown

```diff
@@ -173,6 +174,11 @@ For **data_resource** the following attributes are supported.
 * `type` (Required) - The resource type in which you want to log data events. You can specify only the following value: "AWS::S3::Object", "AWS::Lambda::Function"
 * `values` (Required) - A list of ARN for the specified S3 buckets and object prefixes..
 
+### Insight Selector Arguments
+
+For **insight_selector** the following attributes are supported.
+
+* `insight_type` (Optional) - The type of insights to log on a trail. In this release, only `ApiCallRateInsight` is supported as an insight type.
```

Comment:
Ugh. Removed both lines instead of just one.

```suggestion
* `insight_type` (Optional) - The type of insights to log on a trail. In this release, only `ApiCallRateInsight` is supported as an insight type.

```